portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,8 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function certainOutput() {
```

Comment:
Favor function names which explicitly state what the function is doing. The name `certainOutput()` doesn't tell me much about what is happening. Maybe something like `setFavoriteGame()`?